styles/style.css

```diff
@@ -0,0 +1,95 @@
+.container {
+  background-color: #e5e5e5;
+  width: 389px;
+}
+
+nav {
```

Comment:
[Optional]
My recommendation is, to use CSS classes to style the HTML elements instead of directly styling the HTML elements. Because CSS classes are reusable and easy to maintain. 😃